### DISCUSSION TITLE
el-date-picker日期选择框超出浏览器范围怎么办啊

### DISCUSSION THREAD

**Discussion by Super-12138** [element-plus/element-plus#15316] [Q&A]
el-date-picker日期选择框超出浏览器范围，不知道该怎么处理，找了很多方法都没有用，只有把浏览器缩放调到80%才能自动适应
![08c5416bff79b3b3777704fc537e5e8](https://github.com/element-plus/element-plus/assets/73973689/c7d24ece-c56e-47a2-a692-b8d2a1ac10bb)

**Comment by ryuhangyeong** [comment]
Can you provide a minimal example?

[Here](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6IjxzY3JpcHQgc2V0dXAgbGFuZz1cInRzXCI+XG5pbXBvcnQgeyByZWYsIHZlcnNpb24gYXMgdnVlVmVyc2lvbiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IHZlcnNpb24gYXMgZXBWZXJzaW9uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgRWxlbWVudFBsdXMgfSBmcm9tICdAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZSdcblxuY29uc3QgbXNnID0gcmVmKCdIZWxsbyBXb3JsZCEnKVxuPC9zY3JpcHQ+XG5cbjx0ZW1wbGF0ZT5cbiAgPGgxPnt7IG1zZyB9fTwvaDE+XG4gIDxlbC1pbnB1dCB2LW1vZGVsPVwibXNnXCIgLz5cblxuICA8cD5cbiAgICA8ZWwtaWNvbiBjb2xvcj1cInZhcigtLWVsLWNvbG9yLXByaW1hcnkpXCI+PEVsZW1lbnRQbHVzIC8+PC9lbC1pY29uPlxuICAgIEVsZW1lbnQgUGx1cyB7eyBlcFZlcnNpb24gfX0gKyBWdWUge3sgdnVlVmVyc2lvbiB9fVxuICA8L3A+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiX28iOnt9fQ==)
